core/scripts/vrfv1/main.go

```diff
@@ -84,6 +96,25 @@ func main() {
 		helpers.PanicErr(err)
 		fmt.Printf("Ownerless Consumer: %s TX: %s\n",
 			consumerAddr, helpers.ExplorerLink(chainID, tx.Hash()))
+	case "loadtest-ownerless-consumer-deploy":
+		cmd := flag.NewFlagSet("loadtest-ownerless-consumer-deploy", flag.ExitOnError)
+		coordAddr := cmd.String("coordinator-address", "", "address of VRF coordinator")
+		linkAddr := cmd.String("link-address", "", "address of link token")
+		priceStr := cmd.String("price", "", "the price of each VRF request")
+		helpers.ParseArgs(cmd, os.Args[2:], "coordinator-address", "link-address")
+		price, ok := big.NewInt(0).SetString(*priceStr, 10)
+		if !ok {
+			panic(fmt.Sprintf("failed to parse price amount: %s", *priceStr))
+		}
```

Comment:
I think you can use `decimal.RequireFromString(priceStr).BigInt()` here to save on boilerplate.